wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
Yes, I didn't want to change the current default behavior. But using the current regex I couldn't take any number because it filters them. 

Therefore I've just add a kind of filter in the regex: if the token doesn't match using the first part (the current `\w[\w']`), it tries with the second one (`\w+`). You can test this regex with samples of text if you're not convinced, for instance in this link: https://regex101.com/. But you can see that it's just a modification to let numbers be matched (and they will be removed in the snippet https://github.com/leferrad/word_cloud/blob/number_rendering_support/wordcloud/wordcloud.py#L519-L521 if the argument `render_numbers=False`), but it doesn't cancel the previous regex behavior I guess. 

I'm agree with the fact that it's not fine to "switch" a regex based on the parameter `render_numbers` because it could fork the behavior, so that's the reason why I've just override the current regex. However, if you still uncomfortable with that override, and I can look for another way to achieve this intended feature.